modules/claims_api/app/controllers/concerns/claims_api/disability_compensation_validations.rb

```diff
@@ -411,17 +414,30 @@ def validate_form_526_disability_secondary_disabilities!
     end
 
     def validate_form_526_disability_classification_code!
-      return if (form_attributes['disabilities'].pluck('classificationCode') - [nil]).blank?
-
       form_attributes['disabilities'].each do |disability|
```

Comment:
should we each with index this so we can pass it to the exception on :438? 

so an example would be a response of something like
```disabilities.1.classificationCode```
instead of 
```disabilities.classificationCode```

Reply:
i guess? we werenʼt doing it before, and one imagines the classification code would be a pretty good identifier, but happy to add it while weʼre here.

Reply:
https://github.com/department-of-veterans-affairs/vets-api/pull/19868/commits/209f0f63d3ec400538fc2f34eb065a6b02bee009